application/src/main/java/org/opentripplanner/routing/algorithm/mapping/RouteRequestToFilterChainMapper.java

```diff
@@ -32,7 +34,10 @@ public static ItineraryListFilterChain createFilterChain(
     Instant earliestDepartureTimeUsed,
     Duration searchWindowUsed,
     boolean removeWalkAllTheWayResults,
-    Consumer<PageCursorInput> pageCursorInputSubscriber
+    Consumer<NumItinerariesFilterResults> numItinerariesFilterResultsSubscriber,
+    Consumer<
+      RemoveTransitIfStreetOnlyIsBetterResults
+    > removeTransitIfStreetOnlyIsBetterResultsSubscriber
```

Comment:
Same result vs results.

Reply:
I changed:
- RemoveTransitIfStreetOnlyIsBetterResults to RemoveTransitIfStreetOnlyIsBetterResult
- NumItinerariesFilterResults to NumItinerariesFilterResult